bundle/edu.gemini.ags/src/test/scala/edu/gemini/ags/gems/GemsResultsAnalyzerSpec.scala

```diff
@@ -65,7 +65,7 @@ class GemsResultsAnalyzerSpec extends MascotProgress with Specification with NoT
       }
 
       System.out.println("gems results: size = " + gemsGuideStars.size)
-      gemsGuideStars should have size 164
+      gemsGuideStars should have size 247
```

Comment:
You expect this number to grow when the limits are wider, so this points in the right direction, the bad side is the mascot should be slower given we have more targets